cluster-autoscaler/main.go

```diff
@@ -265,6 +266,7 @@ func createAutoscalingOptions() config.AutoscalingOptions {
 		WriteStatusConfigMap:               *writeStatusConfigMapFlag,
 		StatusConfigMapName:                *statusConfigMapName,
 		BalanceSimilarNodeGroups:           *balanceSimilarNodeGroupsFlag,
+		BalanceSimilarNodeGroupsBy:         *balanceSimilarNodeGroupsByFlag,
```

Comment:
Can you check somewhere that the flag value is actually valid?

Reply:
added code for that